tests/acceptance/pageObjects/FilesPageElement/publicLinksDialog.js

```diff
@@ -43,6 +43,21 @@ module.exports = {
           })
         })
       return Promise.all(promiseList)
+    },
+    /**
+     *
+     * @returns {Promise<string>}
+     */
+    getErrorMessage: async function () {
+      let result
+      await this.getText(
+        'xpath',
+        this.elements.publicLinkContainer.selector + this.elements.errorMessageInsidePublicLinkContainer.selector,
+        function (textValue) {
```

Comment:
Generally, the convention is to name this `result`.
```suggestion
        function (result) {
```

Or, just destructure to get the value:
```suggestion
        function ({value}) {
```